modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -551,7 +551,7 @@ public int getOneToManyRelatedObjectEntriesCount(
 	}
 
 	public Map<String, Object> getSystemModelAttributes(
-			ObjectDefinition objectDefinition, long objectEntryId)
+			ObjectDefinition objectDefinition, long primaryKey)
```

Comment:
Because this method is only called for system objects (for example, `primaryKey` can be the `userId` for the `User` system object) so it doesn't make sense to call it `objectEntryId`.